py/torch_tensorrt/dynamo/aten_tracer.py

```diff
@@ -80,9 +80,6 @@ def trace(
     with unittest.mock.patch(
         "torch._export.DECOMP_TABLE", get_decompositions(experimental_decompositions)
     ):
-        graph_module = export(
-            model, tuple(trace_inputs), constraints=constraints
-        ).module()
+        exp_program = export(model, tuple(trace_inputs), constraints=constraints)
```

Comment:
On line 78 above, consider switching to:
```python
from torch_tensorrt.dynamo._defaults import ENABLE_EXPERIMENTAL_DECOMPOSITIONS
...
    experimental_decompositions = kwargs.get(
        "enable_experimental_decompositions", ENABLE_EXPERIMENTAL_DECOMPOSITIONS
    )
```
So as to use whichever option is enabled by default.

Reply:
done